code/_core/datum/enchantment/_enchantment.dm

```diff
@@ -2,17 +2,18 @@
 	var/name = "Enchantment Name"
 	var/desc = "Enchantment Description"
 	var/enchanting_phrase = "Memes"
-
+	var/max_charge = 0
 	var/charge = 0
 	var/cost = 100
 	var/strength = 1
 
 /enchantment/proc/generate_stats(var/mob/living/caller,var/obj/item/weapon/desired_weapon,var/obj/item/soulgem/desired_soulgem)
 	charge = desired_soulgem.total_charge
+	max_charge = desired_soulgem.total_charge
 	cost = initial(cost) * strength * (1 - caller.get_skill_power(SKILL_MAGIC_ENCHANTING,0,1,1)*0.5)
 	strength = (1 + caller.get_skill_power(SKILL_MAGIC_ENCHANTING,0,1,2)) * (charge/SOUL_SIZE_MYSTIC)*10
 	strength = CEILING(strength,1)
-	name = "[initial(name)] \Roman[strength]"
```

Comment:
Enchantment name with roman strength is now actually handled on load of enchantment/when enchant is applied, so that isn't needed anymore. This fixes the issue where loaded enchantments would not show their strength.